rskj-core/src/main/java/org/ethereum/net/client/ConfigCapabilitiesImpl.java

```diff
@@ -29,26 +29,29 @@
 import java.util.TreeSet;
 
 import static org.ethereum.net.client.Capability.RSK;
+import static org.ethereum.net.client.Capability.SNAP;
 import static org.ethereum.net.eth.EthVersion.fromCode;
 
 /**
  * Created by Anton Nashatyrev on 13.10.2015.
  */
 public class ConfigCapabilitiesImpl implements ConfigCapabilities{
 
-    private final SystemProperties config;
+    private final RskSystemProperties config;
 
     private SortedSet<Capability> allCaps = new TreeSet<>();
 
-    public ConfigCapabilitiesImpl(SystemProperties config) {
+    public ConfigCapabilitiesImpl(RskSystemProperties config) {
         if (config.syncVersion() != null) {
-            EthVersion eth = fromCode(config.syncVersion());
+            EthVersion eth =fromCode(config.syncVersion());
```

Comment:
minor thing but it is better with the space 😓

Reply:
solved